tools/pipeline-runner/cloudbuild.yaml

```diff
@@ -61,10 +61,17 @@ steps:
     - "-c"
     - |-
       source env.txt && \
-      if [ "$_CI_PROJECT" = "all" ]; then
-        run-pipelines.sh || true # see pipeline-result.txt for results
+      if [ -n "$_PR_NUMBER" ]; then
+        git fetch origin
```

Comment:
maybe we should add this diff calculation into `run-pipelines.sh` instead? id like to run this logic from anywhere, not just cloud build

Reply:
^ after this, we can merge

Reply:
moved to separate script